practices/software_development_kit.md

```diff
@@ -0,0 +1,67 @@
+---
+slug: software-development-kit
+stages:
+  - development
+short_description: A SDK is a set of toolchains that allow a developer to write applications for a given platform or allow them to use a specific framework to do so.
+tags:
+  - sdk
+  - software development
+  - software engineering
+  - development
+  - developer experience
+keywords:
+  - product sprint
+  - software development
+  - software engineering
+  - product development
+  - debugging
+  - documentation
+  - libraries
+  - apis
+  - sdk
+---
+
+# Software Development Kit (SDK)
+
+**TL;DR**
+
+It is a set of toolchains that allow a developer to write applications for a given platform or allow them to use a specific framework to do so. You have what you need to get the job done, and what's included in your kit toolbox, varies from platform to platform.
+
+## What is an SDK
+
+A Software Development Kit (SDK) is the toolbox of the software development world. But instead of tools like screwdrivers and pliers, SDKs contain libraries and Application Programming Interfaces (APIs) that make it easier to create applications for specific software and/or hardware.
```

Comment:
Similar to _A Software Development Kit (SDK) is the toolbox of the software development world. But instead of tools like screwdrivers and pliers, SDKs contain libraries and APIs (Application Programming Interfaces) that make it easier to create applications for specific software and hardware._ from https://www.freecodecamp.org/news/whats-an-sdk-software-development-kits-explained-in-plain-english/

Reply:
Trying to rewrite this part as soon as able :)